wlauto/instrumentation/daq/__init__.py

```diff
@@ -158,6 +159,11 @@ class Daq(Instrument):
                   to facillitate syncing kernel trace events to DAQ power
                   trace.
                   """),
+        Parameter('merge_channels', kind=bool, default=False,
+                  description="""
+                  If set to true, channels with consecutive letter suffixes will be summed.
+                  e.g. If you have channels A7a, A7b, A7c, A15a, A15b they will be summed to A7, A15
```

Comment:
Rather than doing it this way, and constraining the naming of individual channels, why not specify the mapping explicitly, e.g.

    merged_channels:
                 a53: [a53dvfs, a53dram]
                 a72: [a72dvfs, a72dram]